src/LoggerListeners.cpp

```diff
@@ -15,13 +15,13 @@
 	{
 	protected:
 
-		virtual void SetLogColour(cLogger::eLogLevel a_LogLevel) = 0;
+		virtual void SetLogColour(eLogLevel a_LogLevel) = 0;
 		virtual void SetDefaultLogColour() = 0;
 
-		virtual void Log(AString a_Message, cLogger::eLogLevel a_LogLevel) override
+		virtual void Log(std::string_view a_Message, eLogLevel a_LogLevel) override
 		{
 			SetLogColour(a_LogLevel);
-			fputs(a_Message.c_str(), stdout);
+			fwrite(a_Message.data(), 1, a_Message.size(), stdout);
```

Comment:
oooh systems with 4 byte chars ain't gonna like that 1 ;)

sizeof(string_view::value_type) maybe

Reply:
Or cout << a_Message

Reply:
`sizeof(char) == 1` by definition, you're thinking of `CHAR_BIT`.

Reply:
darn. well put sizeof(char) in there anyway to be more obvious what the argument is